joern-cli/frontends/javasrc2cpg/src/main/scala/io/joern/javasrc2cpg/JavaSrc2Cpg.scala

```diff
@@ -50,4 +50,22 @@ object JavaSrc2Cpg {
       new JavaTypeHintCallLinker(cpg)
     )
   }
+
+  def showEnv(): Unit = {
+    val value =
+      JavaSrcEnvVar.values.foreach { envVar =>
+        val currentValue = Option(System.getenv(envVar.name)).getOrElse("<unset>")
+        println(s"${envVar.name}:")
+        println(s"- Description  : ${envVar.description}")
+        println(s"- Current value: $currentValue")
+      }
+  }
+
+  enum JavaSrcEnvVar(val name: String, val description: String) {
```

Comment:
This is perhaps excessive for a single environment variabl, but I also think that, if environment variables are going to be used for configuration, they should be documented somehow and easily discoverable. I plan to extend this idea to the rest of joern (I know at least php2cpg uses some environment variables as well), but put this here since it's currently limited to javasrc.